Heat_method_3/package_info/Heat_method_3/dependencies

```diff
@@ -18,3 +18,5 @@ Random_numbers
 STL_Extension
 Solver_interface
 Stream_support
+Weights
+Polygon
```

Comment:
Where does this dependency come from?

Reply:
I fixed that. It does not depend on the Polygon anymore.